src/deepEqual.js

```diff
@@ -3,10 +3,8 @@
 /**
  * Implement deepEqual function:
  *
- * Function compares 2 values
- * for primitives it compares types and values of given params
- * for objects it checks all fields and compare primitive values
- * inside all fields. Nested objects are compared too.
+ * Non object types are compared with ===
```

Comment:
nit: don't put line breaks in random places. Line break within 1 paragrapth should be when we reach 80 char limit. In this case line break should be after own. 

* Non object types are compared with ===. Objects are equal if all the own 
* enumerable properties are equal

Reply:
Why do you think it is better?
For me it is easier to understand that it is another sentence if I put it on it's own new line
rather then search for the dot in the middle of long line

Reply:
In writings (documents, emails etc.) we use '.' to separate different sentences. jsdocs aren't different from that perspective. 

example of google documentation: 
https://github.com/google/closure-library/blob/master/closure/goog/collections/sets.js#L28

example of facebook documentation:
https://github.com/facebook/react/blob/master/scripts/jest/matchers/reactTestMatchers.js#L7

They all do line breaks at 80 characters. Not at sentences.

Reply:
Here is an opposite example from the next file to the file you gave from jest
https://github.com/facebook/react/blob/master/scripts/jest/matchers/toWarnDev.js#L48

Reply:
And here is an example from google closure repo
https://github.com/google/closure-library/blob/master/closure/goog/color/color.js#L38

Reply:
So I would not say it is so strict to follow close to 80 length
But I agree that in most cases it males sense

Reply:
first example is weird. they do ',' and then capital case. More of a example of bad documentation.

the second example is an example of bullet points. Yes, bullet points we do from new lines. 

https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping
`but this horizontal alignment is discouraged.`

Reply:
Like it!
Let's add bullet points here for each of 2 options